Tools/clinic/clinic.py

```diff
@@ -294,7 +294,7 @@ def flush():
     flush()
     return tuple(version)
 
-def version_comparitor(version1, version2):
+def version_comparator(version1, version2):
```

Comment:
Not sure who our current clinic maintainer would be, but someone should check that all references have been handled.

Reply:
Ping @erlend-aasland who has been checking it recently.

Reply:
Thanks for the ping, Hugo. The clinic changes are technically ok. I would not bother fixing this myself, but since the PR is here already, I guess its fine.